waymo_raw_data_to_argoverse.py

```diff
@@ -107,22 +135,36 @@ def main(save_images: bool, save_poses: bool, save_calibration: bool):
 			frame.ParseFromString(bytearray(data.numpy()))
 			# discovered_log_id = '967082162553397800_5102_900_5122_900'
 			assert log_id == frame.context.name
-			
 			# Frame start time, which is the timestamp of the first top lidar spin
 			# within this frame, in microseconds
 			timestamp_ms = frame.timestamp_micros
 			timestamp_ns = int(timestamp_ms * 1000) # to nanoseconds
 			SE3_flattened = np.array(frame.pose.transform)
 			city_SE3_egovehicle = SE3_flattened.reshape(4,4)
 			if save_poses:
-				dump_pose(city_SE3_egovehicle, timestamp_ns, log_id)
-
+				dump_pose(city_SE3_egovehicle, timestamp_ns, log_id, ARGO_WRITE_DIR)
+			# Reading lidar data and saving it point cloud format
+			(range_images,
+			camera_projections,
+			range_image_top_pose) = frame_utils.parse_range_image_and_camera_projection(frame)
+			points_ri2, cp_points_ri2 = frame_utils.convert_range_image_to_point_cloud(
```

Comment:
is this using the first return?

Reply:
I changed it to use first return by default, but now the user can specify using `argparse`.